pkg/hostagent/hostagent.go

```diff
@@ -317,6 +317,18 @@ func (a *HostAgent) startHostAgentRoutines(ctx context.Context) error {
 	if err := a.waitForRequirements(ctx, "essential", a.essentialRequirements()); err != nil {
 		mErr = multierror.Append(mErr, err)
 	}
+	if *a.y.SSH.ForwardAgent {
+		faScript := `#!/bin/bash
+set -eux -o pipefail
```

Comment:
Does this work for Alpine?

Reply:
Yes, it appears to work with this example.

```
# This example requires Lima v0.7.0 or later.
images:
- location: "https://github.com/lima-vm/alpine-lima/releases/download/v0.2.26/alpine-lima-std-3.17.0-x86_64.iso"
  arch: "x86_64"
  digest: "sha512:93bf8d52ad2b3a6ef401a1a565f67e4f06f8022ec8963cad36cac4a25253f056ac0b755d1ed56b8993b0261440e0d5d38bad15b271864ed33cd3b02272738672"
- location: "https://github.com/lima-vm/alpine-lima/releases/download/v0.2.26/alpine-lima-std-3.17.0-aarch64.iso"
  arch: "aarch64"
  digest: "sha512:aacd3a9e7a3231553bcee879c6770b379afbfad3dcfac7f274a80d6c490faf33f3c23b6130efa881f57c0c5235ced2877e2f45e3cdf4fefff11531d6023ae214"

mounts:
- location: "~"
- location: "/tmp/lima"
  writable: true

# The built-in containerd installer does not support Alpine currently.
# Hint: use the "rd" ISO instead of the "std" ISO to enable containerd: https://github.com/lima-vm/alpine-lima/releases/
containerd:
  system: false
  user: false

ssh:
  forwardAgent: true
```

Validation.

```
╰─ _output/bin/limactl shell default

lima-default:/Users/ryancurrah/git/github.com/ryancurrah/lima$ ls -l /run/host-services/ssh-auth.sock
lrwxrwxrwx    1 root     root            30 Feb  2 01:53 /run/host-services/ssh-auth.sock -> /tmp/ssh-XXXXpKBFdf/agent.2615

lima-default:/Users/ryancurrah/git/github.com/ryancurrah/lima$ ssh-add -l
4096 SHA256:....Od0ivA ryan@currah.ca (RSA)
```

Reply:
> Does this work for Alpine?

Yes, it does; we use it in the boot scripts too, e.g. https://github.com/lima-vm/lima/blob/f7e7addab557da560da7146a829c32c59e939a33/pkg/cidata/cidata.TEMPLATE.d/boot/05-lima-disks.sh#L3